Simplenote/AboutViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AboutViewController.swift
+//  Simplenote
+//
+
+import Cocoa
+
+class AboutViewController: NSViewController {
+    let blogUrl =       "https://simplenote.com/blog"
+    let twitterUrl =    "https://twitter.com/simplenoteapp"
+    let appsUrl =       "https://simplenote.com"
+    let githubUrl =     "https://github.com/automattic/simplenote-macos"
+    let hiringUrl =     "https://automattic.com/work-with-us"
+    let privacyUrl =    "https://simplenote.com/privacy/"
+    let termsUrl =      "https://simplenote.com/terms/"
+    
+    @IBOutlet var blogLabel:        SPAboutTextField?
```

Comment:
IBOutlets are usually set up as Implicitly Unwrapped Optionals (i _suspect_ the IUO was even made up for this scenario!). I think, mostly, the reason is you want to crash if there's a missing outlet (and it's easier than adding asserts).